.travis.yml

```diff
@@ -41,3 +56,6 @@ script:
       cd ../test/unit_tests/cuda;
       make;
     fi
+
+after_script:
+  - script/travis/print_valgrind_log.sh
```

Comment:
Here should be `../script/travis/print_valgrind_log.sh` as we execute the script from `build` directory.